src/Bynder/Api/Impl/AbstractRequestHandler.php

```diff
@@ -34,5 +41,12 @@ function ($response) {
         );
     }
 
+
+    public function sendRequestAsyncRawResponse($requestMethod, $uri, $options = [])
```

Comment:
This method isn't used?

Reply:
Yeah I think I made this to get the header from the `finalise_api` response much like the python-sdk(as I was emulating my work from there) but I used the `response->getHeader()` instead and never ended up using it. I will take this out which should also resolve some other comments @githuib had as well.